core/app/models/spree/order_contents.rb

```diff
@@ -6,6 +6,23 @@ def initialize(order)
       @order = order
     end
 
+    # Add a line items to the order if there is inventory to do so
+    # and populate Subscriptions and Promotions
+    #
+    # @params [Spree::Variant] :variant The variant the line_item should
+    #   be associated with
+    # @params [Integer] :quantity The line_item quantity
+    # @param [Hash] :options Options for the adding proccess
+    #   Valid options:
+    #     shipment: [Spree::Shipment] LineItem target shipment
+    #     stock_location_quantities:
+    #       stock_location_id: The stock location to source from
+    #       quanity: The quantity of stock to source from the location
+    #     id: Line item options id
+    #     variant_id: Line item options variant_id
+    #     quantity: line item options quantity
```

Comment:
We should probably not document them since they may have an effect but will likely produce strange behaviour